edubots.bib

```diff
@@ -7,8 +7,8 @@ @InProceedings{tsivitanidouusers
 @misc{differchat, title={Differ - Helping you connect with other new students}, url={https://www.differ.chat/}, journal={Differ Chat} (14/06/2021)} 
 @misc{lancasteruniversity_2019, title={Lancaster University launch pioneering chatbot companion for students}, url={https://www.lancaster.ac.uk/news/lancaster-university-launch-pioneering-chatbot-companion-for-students}, journal={Lancaster University}, year={2019}, month={Mar} (14/06/2021)} 
 @misc{meetholly, title={Meet Holly}, url={https://www.dur.ac.uk/study/askus/holly/}, journal={Study : Meet Holly - Durham University}(14/06/2021)} 
-@unpublished{abbas2021onlinechat , author = { Abbas, N., Whitfield, J., Atwell, E., Bowman, H., Pickard, T. and Walker, A.}, title = { Online Chat and Chatbots to Enhance Mature Student Engagement in Higher Education}, note = {Manuscript submitted for publication}, year = {2021} }
-@inproceedings{abbas2021Surveys , author = { Abbas, N., Pickard, T., Atwell, E. and Walker, A.}, title = { University Student Surveys using Chatbots: Artificial Intelligence Conversational Agents }, booktitle={ HCI International Conference 2021}, series={ Learning and Collaboration Technologies- Chatbots in Education }, Publisher = {Springer}, year = {2021}, note = {(in press)} }
+@unpublished{abbas2021onlinechat , author = { Abbas, N., Whitfield, J., Atwell, E., Bowman, H., Pickard, T. and Walker, A.}, title = { Online Chat and Chatbots to Enhance Mature Student Engagement in Higher Education}, note = {Manuscript submitted for publication}, year = {2021a} }
+@inproceedings{abbas2021Surveys , author = { Abbas, N., Pickard, T., Atwell, E. and Walker, A.}, title = { University Student Surveys using Chatbots: Artificial Intelligence Conversational Agents }, booktitle={ HCI International Conference 2021}, series={ Learning and Collaboration Technologies- Chatbots in Education }, Publisher = {Springer}, year = {2021b}, note = {(in press)} }
```

Comment:
I'm afraid the year needs to be just a number. Don't worry about the reference, that will be set automatically...